lib/core/network/models/transaction_model.dart

```diff
@@ -38,9 +38,7 @@ sealed class TransactionModel extends Equatable {
     required Map<String, dynamic> data,
   }) {
     return switch (actionName) {
-      'transfer'
-          when account == _hyphaTokenAccount || account == _hyphaWrapTokenAccount || account == _systemTokenAccount =>
```

Comment:
when account == _hyphaTokenAccount || account == _hyphaWrapTokenAccount || account == _systemTokenAccount
 
 
 Is this needed?? 
 
 
 Basically this is never true when it call it for my account.

Reply:
yeah because we recognize our own wrap token accounts

This is only identifying a transaction as a transfer

system token = TLOS or EOS
hypha token = hypha.hypha
wrap token = this isn't working yet but I think we should keep it there

Reply:
ok so a transfer is any actions that includes 'transfer' as the action name

Later on we decide which one to show or not. But the model is not the one dictating this, the UI is. 

The models just models data.